Doc/c-api/frame.rst

```diff
@@ -41,6 +41,17 @@ See also :ref:`Reflection <reflection>`.
    .. versionadded:: 3.9
 
 
+.. c:function:: PyCodeObject* PyFrame_GetLocals(PyFrameObject *frame)
```

Comment:
It returns a dictionary, no?

```suggestion
.. c:function:: PyDictObject* PyFrame_GetLocals(PyFrameObject *frame)
```

Reply:
It should return `PyObject *` not `PyDictObject *`

Reply:
Oh I didn't check :-)

Reply:
Definitely not a `PyCodeObject*`. Thanks for spotting that.